components/chainhook-sdk/src/chainhooks/mod.rs

```diff
@@ -1,3 +1,6 @@
 pub mod bitcoin;
 pub mod stacks;
 pub mod types;
+
+#[cfg(test)]
+pub mod tests;
```

Comment:
```suggestion
pub mod tests;

```
(the diff editor was complaining that it's missing a newline at the end of the file)